testing/e2e/infra/types.go

```diff
@@ -81,6 +83,7 @@ type cert interface {
 type Provisioned struct {
 	Name              string
 	Cluster           cluster
+	SPCluster         cluster
```

Comment:
Cluster should have a consistent interface. We shouldn't add SPCluster here. Make the SPCluster fit under the Cluster inteface

Reply:
done